tdrs-backend/tdpservice/settings/common.py

```diff
@@ -194,7 +194,7 @@ class Common(Configuration):
 
     # Logging
     # set level as 'INFO' if env var is not set
-    LOGGING_LEVEL = os.getenv('LOGGING_LEVEL', 'INFO')
+    LOGGING_LEVEL = "WARNING"#os.getenv('LOGGING_LEVEL', 'INFO')
```

Comment:
Remnant from debugging locally? Surely this wasn't in the ACs from original ticket.

Reply:
Yep, debugging remnant. Will fix